uikit-dz/uikit-dz/Cells/RecomendationTableViewCell.swift

```diff
@@ -0,0 +1,295 @@
+// RecomendationTableViewCell.swift
+// Copyright © RoadMap. All rights reserved.
+
+import UIKit
+
+// Ячейка для рекомендаций
+final class RecomendationTableViewCell: UITableViewCell {
+    // перечисление для констант
+    enum Constant: String {
+        case recommend = "Рекомендуем вам"
+        case all = "Все"
+        case crimea
+        case crimea082 = "crimea_082"
+        case xmark
+        case subscribe = "Подписаться"
+        case identifier = "RecomendationTableViewCell"
+    }
+
+    // MARK: - Public static properties
+
+    static let identifier = Constant.identifier.rawValue
+
+    // MARK: - Private ui elements
+
+    private let recomendTitleLabel: UILabel = {
+        let label = UILabel()
+        label.text = Constant.recommend.rawValue
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.font = .systemFont(ofSize: 12)
+        return label
+    }()
+
+    private let allButton: UIButton = {
+        let button = UIButton()
+        button.translatesAutoresizingMaskIntoConstraints = false
+        button.setTitle(Constant.all.rawValue, for: .normal)
+        button.setTitleColor(.blue, for: .normal)
+        button.titleLabel?.font = .systemFont(ofSize: 12)
+        return button
+    }()
+
+    private let myscrollView: UIScrollView = {
+        let scrollView = UIScrollView()
+        scrollView.translatesAutoresizingMaskIntoConstraints = false
+        scrollView.showsVerticalScrollIndicator = false
+        scrollView.showsHorizontalScrollIndicator = false
+        scrollView.backgroundColor = #colorLiteral(red: 0.8483793139, green: 0.889921844, blue: 0.944878161, alpha: 1)
+        return scrollView
+    }()
+
+    private let myView1: UIView = {
```

Comment:
```suggestion
    private let myView1: UIView = {
```
my не понятно что за вью